hudi-spark-datasource/hudi-spark3.3.x/src/main/antlr4/org/apache/hudi/spark/sql/parser/HoodieSqlBase.g4

```diff
@@ -38,3 +90,944 @@ statement
     | REFRESH INDEX identifier ON TABLE? tableIdentifier               #refreshIndex
     | .*?                                                              #passThrough
     ;
+
+identifier
+    : strictIdentifier
+    | {!SQL_standard_keyword_behavior}? strictNonReserved
```

Comment:
Can the template be extended in some way so that the definitions of common ids and keywords can be shared.